scrapy/exporters.py

```diff
@@ -128,33 +129,52 @@ def __init__(self, file, **kwargs):
             self.encoding = 'utf-8'
         self.xg = XMLGenerator(file, encoding=self.encoding)
 
+    def _beautify_newline(self):
+        if self.indent_width:
```

Comment:
The link is not pointing to the previous commit 😕 
Anyway, the line was `self.indent = settings.getint('FEED_EXPORT_INDENT') or None`, so `self.indent` ended up being `None` if `settings['FEED_EXPORT_INDENT']` was `0`